midpoint/features/planned/scheduled-single-run-tasks.adoc

```diff
@@ -0,0 +1,20 @@
+---
+midpoint-feature: task-management
+doc-type: plan
+---
+= Scheduled Single-Run Tasks
+:page-planned: true
+:page-upkeep-status: yellow
+:page-toc: top
+:page-description: Scheduling single-run tasks is a possible feature for future releases of midPoint.
+:page-keywords: tasks, scheduling
+
+== Motivation
+
+You can xref:/midpoint/reference/tasks/task-manager/#task-scheduling[schedule recurring tasks] in midPoint already.
+However, it's not possible to schedule single-run tasks, such as an import from a resource.
```

Comment:
Technically you are right but it can be bypassed by setting cron pattern to a fixed date&time.